### PR TITLE
BFW-1691 ESP_Update enhancement

### PR DESCRIPTION
**Pull request by zoltan-l**
Pull requests :
1. Adds a boot_v1.7. bin file to update - name harmonization with [espressive ](https://www.espressif.com/en/support/download/at?keys=&field_type_tid%5B%5D=14&field_type_tid%5B%5D=799) file naming
2. adds ESP subfolder for ESP_update files storage
3. add user warning during update

### CODE REVIEW

File: src/gui/screen_menu_esp_update.cpp [hunk 1]
```diff
@@ -99,12 +99,12 @@ ScreenMenuESPUpdate::ScreenMenuESPUpdate()
...
-          { .address = BLANK1_ADDRESS, .filename = "/blank.bin", .size = 0 },
-          { .address = BLANK2_ADDRESS, .filename = "/blank.bin", .size = 0 },
-          { .address = INIT_DATA_ADDRESS, .filename = "/esp_init_data_default_v08.bin", .size = 0 },
-          { .address = BLANK3_ADDRESS, .filename = "/blank.bin", .size = 0 } } })
+    , firmware_set({ { { .address = BOOT_ADDRESS, .filename = "/ESP/boot_v1.7.bin", .size = 0 },
+          { .address = USER_ADDRESS, .filename = "/ESP//user1.1024.new.2.bin", .size = 0 },
+          { .address = BLANK1_ADDRESS, .filename = "/ESP//blank.bin", .size = 0 },
+          { .address = BLANK2_ADDRESS, .filename = "/ESP//blank.bin", .size = 0 },
+          { .address = INIT_DATA_ADDRESS, .filename = "/ESP//esp_init_data_default_v08.bin", .size = 0 },
+          { .address = BLANK3_ADDRESS, .filename = "/ESP//blank.bin", .size = 0 } } })
```
Comment (vorner) [inline src/gui/screen_menu_esp_update.cpp:107]: Why do you have the double slashes there? While it works (duplicate `/` are generally ignored), it shouldn't be needed.

File: src/gui/screen_menu_esp_update.cpp [hunk 2]
```diff
@@ -160,7 +160,7 @@ void ScreenMenuESPUpdate::windowEvent(EventLock /*has private ctor*/, window_t *
             esp_loader_connect_args_t config = ESP_LOADER_CONNECT_DEFAULT();
             _dbg("ESP boot connect");
             if (ESP_LOADER_SUCCESS == esp_loader_connect(&config)) {
-                help.SetText(_("Successfully connected to ESP module and switched into UART bootloader."));
+                help.SetText(_("Successfully connected to ESP, do not swith the Printer off nor remove the Flash disk."));
```
Comment (vorner) [inline src/gui/screen_menu_esp_update.cpp:163]: Minor suggestion: Split into two sentences? `Successfully connected to ESP. Do not switch…`?

File: src/gui/screen_menu_esp_update.cpp [hunk 3]
```diff
@@ -212,6 +212,7 @@ void ScreenMenuESPUpdate::windowEvent(EventLock /*has private ctor*/, window_t *
         }
         case esp_upload_action::Reset:
             _dbg("ESP finished flahing");
+            help.SetText(_("ESP succesfully flash, WiFI initiation started."));
```
Comment (vorner) [inline src/gui/screen_menu_esp_update.cpp:215]: Past tense (`flashed`)?

**Review by vorner — CHANGES_REQUESTED** [review]
Mostly fine, with just minor nitpicking.

File: src/gui/screen_menu_esp_update.cpp (hunk 3 above)
Comment (zoltan-l) [reply src/gui/screen_menu_esp_update.cpp]: this is visible only for debugger, but no problem

File: src/gui/screen_menu_esp_update.cpp (hunk 2 above)
Comment (zoltan-l) [reply src/gui/screen_menu_esp_update.cpp:163]: ok

File: src/gui/screen_menu_esp_update.cpp (hunk 1 above)
Comment (zoltan-l) [reply src/gui/screen_menu_esp_update.cpp:107]: stupidity corrected

**Review by vorner — APPROVED** [review]
Thank you :+1: 

(Please use the squash method of merging, as discussed)

**Pull request merged**